tsconfig.prod.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "./tsconfig",
+  "exclude": ["node_modules", "dist", "**/*.stories.*", "src/tests"]
```

Comment:
so the `declaration: true` emits the `.d.ts` files - this is the package contents after compilation:

```
➜  formio-renderer git:(chore/package-building) tree dist/esm/
dist/esm/
├── components
│   ├── charcount
│   │   ├── charcount.component.d.ts
│   │   ├── charcount.component.js
│   │   ├── charcount.component.js.map
│   │   ├── index.d.ts
│   │   ├── index.js
│   │   └── index.js.map
│   ├── columns
│   │   ├── columns.component.d.ts
│   │   ├── columns.component.js
│   │   ├── columns.component.js.map
│   │   ├── index.d.ts
│   │   ├── index.js
│   │   └── index.js.map
│   ├── component
│   │   ├── component.component.d.ts
│   │   ├── component.component.js
│   │   ├── component.component.js.map
│   │   ├── index.d.ts
│   │   ├── index.js
│   │   └── index.js.map
│   ├── content
│   │   ├── content.component.d.ts
│   │   ├── content.component.js
│   │   ├── content.component.js.map
│   │   ├── index.d.ts
│   │   ├── index.js
│   │   └── index.js.map
│   ├── description
│   │   ├── description.component.d.ts
│   │   ├── description.component.js
│   │   ├── description.component.js.map
│   │   ├── index.d.ts
│   │   ├── index.js
│   │   └── index.js.map
│   ├── errors
│   │   ├── errors.component.d.ts
│   │   ├── errors.component.js
│   │   ├── errors.component.js.map
│   │   ├── index.d.ts
│   │   ├── index.js
│   │   └── index.js.map
│   ├── index.d.ts
│   ├── index.js
│   ├── index.js.map
│   ├── label
│   │   ├── index.d.ts
│   │   ├── index.js
│   │   ├── index.js.map
│   │   ├── label.component.d.ts
│   │   ├── label.component.js
│   │   └── label.component.js.map
│   └── textfield
│       ├── index.d.ts
│       ├── index.js
│       ├── index.js.map
│       ├── textfield.component.d.ts
│       ├── textfield.component.js
│       └── textfield.component.js.map
├── fixtures
│   ├── formio
│   │   ├── formio-columns.d.ts
│   │   ├── formio-columns.js
│   │   ├── formio-columns.js.map
│   │   ├── formio-example.d.ts
│   │   ├── formio-example.js
│   │   ├── formio-example.js.map
│   │   ├── formio-length.d.ts
│   │   ├── formio-length.js
│   │   ├── formio-length.js.map
│   │   ├── formio-pattern.d.ts
│   │   ├── formio-pattern.js
│   │   ├── formio-pattern.js.map
│   │   ├── formio-required.d.ts
│   │   ├── formio-required.js
│   │   ├── formio-required.js.map
│   │   ├── index.d.ts
│   │   ├── index.js
│   │   └── index.js.map
│   ├── index.d.ts
│   ├── index.js
│   └── index.js.map
├── index.d.ts
├── index.js
├── index.js.map
└── lib
    ├── renderer
    │   ├── index.d.ts
    │   ├── index.js
    │   ├── index.js.map
    │   ├── renderer.d.ts
    │   ├── renderer.js
    │   └── renderer.js.map
    └── validation
        ├── index.d.ts
        ├── index.js
        ├── index.js.map
        ├── validate.d.ts
        ├── validate.js
        ├── validate.js.map
        ├── validationerror.d.ts
        ├── validationerror.js
        ├── validationerror.js.map
        └── validators
            ├── index.d.ts
            ├── index.js
            ├── index.js.map
            ├── maxlength.d.ts
            ├── maxlength.js
            ├── maxlength.js.map
            ├── minlength.d.ts
            ├── minlength.js
            ├── minlength.js.map
            ├── pattern.d.ts
            ├── pattern.js
            ├── pattern.js.map
            ├── required.d.ts
            ├── required.js
            └── required.js.map
```
I think you *do* want to ship the types for downstream projects so that they also have the correct type information. Eventually we'll make the SDK itself also typescript. Or am I misunderstanding the question?